File: aspnet-core/src/HotelApp.Application.Contracts/Hotels/HotelDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace HotelApp.Hotels
{

    public class HotelDto : AuditedEntityDto<Guid>
    {
        public string Name { get; set; }
        public string ShortDescription { get; set; }
        public string FullDescription { get; set; }

        public bool IsEnabled { get; set; }

        public bool ShowOnHomePage { get; set; }

        public bool AllowCustomerReviews { get; set; }
        public string DefaultPicture { get; set; }

        public HotelAddressDto Address { get; set; }
        public HotelLocationDto Location { get; set; }

        public HotelTypeDto HotelType { get; set; }
    }
}

File: angular/src/app/hotel/models/hotel-dto.ts


import { AuditedEntityDto } from '@abp/ng.core';

export class HotelDto extends AuditedEntityDto<string> {
  name: string;
  shortDescription: string;
  fullDescription: string;
  isEnabled: boolean;
  defaultPicture: string;
  displayAddress: string;
  displayLocation: string;
  hotelType: string;
  lastModificationTime?: string;
  lastModifierId?: string;
  creationTime: string;
  creatorId?: string;
  id: string;

  constructor(initialValues: Partial<HotelDto> = {}) {
    super(initialValues);
  }
}
